include/z64pause.h

```diff
@@ -174,12 +174,12 @@ typedef struct PauseContext {
     /* 0x01E8 */ u16 pageIndex; // "kscp_pos"
     /* 0x01EA */ u16 pageSwitchTimer;
     /* 0x01EC */ u16 savePromptState;
-    /* 0x01F0 */ f32 unk_1F0;
-    /* 0x01F4 */ f32 unk_1F4;
-    /* 0x01F8 */ f32 unk_1F8;
-    /* 0x01FC */ f32 unk_1FC;
-    /* 0x0200 */ f32 unk_200;
-    /* 0x0204 */ f32 unk_204; // "angle_s"
+    /* 0x01F0 */ f32 promptOffsetDepth; // Offset position of the prompt away from the camera
+    /* 0x01F4 */ f32 itemPageRoll; // Rotation of the item page around its local horizontal/sideways axis
```

Comment:
based on the comment this should be `itemPagePitch` right? roll would be around its Z axis. that said i'm not a huge fan of that either. `itemPageXRot`?

Reply:
You're right it's a pitch lol idk how I used roll for this all this time